src/postgraphql/withPostGraphQLContext.js

```diff
@@ -0,0 +1,45 @@
+import { $$pgClient } from '../postgres/inventory/pgClientFromContext'
+import debugPgClient from './http/debugPgClient'
+import setupPgClientTransaction from './setupPgClientTransaction'
+
+/*
+
+This is intended to be called similar to this:
+
+    const result = await withPostGraphQLContext(options, { pgPool, jwtToken }, async context => (
+      await graphql(
+        schema,
+        query,
+        null,
+        {  ...context },
+        variables,
+        operationName,
+      )
+    ))
+
+*/
+
+export default async function withPostGraphQLContext(options, {pgPool, jwtToken}, functionToRun) {
```

Comment:
@calebmer  I'd rather you added types, I'm not experienced enough with TypeScript to declare them idiomatically. I'll learn a lot by seeing how you do them, too 👍